### PR TITLE
Password management and best practices

### PR DESCRIPTION
**Pull request by WilliamLoy**

### CODE REVIEW

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-a39b4b73de/how-to/). :bow:

**Comment by stephamon** [comment]
Hey @DonNateR here are my suggestions. Let me know if you have any questions!

[Password.management.and.best.practices_SFedit.docx](https://github.com/rackerlabs/rackspace-how-to/files/1492242/Password.management.and.best.practices_SFedit.docx)
[Password.management.and.best.practices_SFedit.pdf](https://github.com/rackerlabs/rackspace-how-to/files/1492243/Password.management.and.best.practices_SFedit.pdf)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-9f0ef0d4d4/how-to/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-742700d6fa/how-to/).

File: content/rackspace-email/password-management-and-best-practices.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+For more information about prerequisite terminology, see [Cloud Office support terminology](/how-to/cloud-office-support-terminology).
+
+
+### Common password attack types
+
+The following methods are the most common used by attackers to attempt to compromise your accounts:
+
+#### Phishing 
+
+Phishing is the act of impersonating a legitimate entity to obtain sensitive information from users such as usernames, passwords, credit card numbers, and so on. In email, phishing scams commonly utilize a method of email impersonation called [spoofing](/how-to/email-spoofing-explained). A successful phishing attack is particularly damaging because you have volunteered your current credentials to the scammer and they now have access.
```
Comment (catlook) [inline content/rackspace-email/password-management-and-best-practices.md]: ...  scammer, and they ...

File: content/rackspace-email/password-management-and-best-practices.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+Use the following guidelines to protext your passwords from phishing attempts:
```
Comment (catlook) [inline content/rackspace-email/password-management-and-best-practices.md]: ... to protect your ...

File: content/rackspace-email/password-management-and-best-practices.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+#### Dictionary Attack 
+
+Hackers try a list of passwords against a username in hopes that the user has used an easy-to-guess password.
+
+To prevent hackers from guessing your passwords create a unique password. Using a common password makes you a prime target for this attack and you should strengthen your password. See [Password best practices](#password-best-practices) for guidance.
+
+#### Password Reset Protocol Attack 
+
+Password reset protocols are typically based off of alternate contact information like phone numbers or email address. If a hacker has the information to reset your password, they have no need for you current password.
```
Comment (catlook) [inline content/rackspace-email/password-management-and-best-practices.md]: recommend "based on" instead of "based off of"

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-b0311e0192/how-to/)

**Pull request merged**